weather_mv/README.md

```diff
@@ -411,6 +414,25 @@ weather-mv ee --uris "gs://your-bucket/*.grib" \
            --ee_max_concurrent 10
 ```
 
+Custom Band names:
+
+```bash
+weather-mv ee --uris "gs://your-bucket/*.tif" \
+           --asset_location "gs://$BUCKET/assets" \ # Needed to store assets generated from *.tif
+           --ee_asset "projects/$PROJECT/assets/test_dir" \
+           --band_names_mapping "filename.json"
+```
+
+Getting initialization and forecast/end date-time from the filename:
+
+```bash
+weather-mv ee --uris "gs://your-bucket/*.tif" \
```

Comment:
Can you add an example regex for a file path?

Reply:
Do you mean to add the regex for the file path (--uris flag) or for the initialization_time_regex / forecast_time_regex flags ?

Reply:
Resolved. Have added a sample example for a file.